src/markdown-pages/automate-workflows/error-inbox/set-up-error-inbox.mdx

```diff
@@ -0,0 +1,199 @@
+![view ignored error groups](../../../images/error-inbox/view-ignored-errors.png)
+
+Your errors inbox now show all the ignored error groups.
+![ignored error groups](../../../images/error-inbox/ignored-error-groups.png)
+
+## Assign errors
+Change the errors inbox filter to show all the unresolved errors.
+![unresolved error groups](../../../images/error-inbox/unresolved-errors.png)
+
+Oops! you were told that somebody fixed **pika.exceptions:ChannelWrongStateError** error but you still see it under unresolved errors. You don't know who fixed this error previously and there's no way for you to find that out. To avoid such situation in future, assign the error from inside errors inbox.
+
+Click the human icon on the far right side of the error group.
+![assign error](../../../images/error-inbox/assign-error.png)
+
+Enter the email address of the person you want to assign error to, and click **+**. 
+![assign error via email](../../../images/error-inbox/assign-error-via-email.png)
+
+The error is now assigned and its now easier for you to keep a track of who worked on the error.
+
+<Callout variant='important'>
+
+Currently assigning an error group to a user does not send a notification. Notifications of assignment and changes to error groups will be coming soon.
+</Callout>
+
+## Connect an inbox to Jira
+Connect errors inbox to jira to qucikly and easily create tickets containing error details and link to the stack trace. It allows you to collaborate and resolve errors faster.
+
+Connect your errors inbox to jira:
+<Steps>
+<Step>
+
+Click on the Jira integration icon on the far right side of the error group you want to connect to Jira.
+![jira integration](../../../images/error-inbox/jira-integration.png)
+</Step>
+<Step>
+
+Click **Add JIRA Workspace** from the dropdown.
+![add jira workspace](../../../images/error-inbox/add-jira-workspace.png)
+</Step>
+<Step>
+
+Fill in the fields on the next page and click **Test connection** to ensure that your details are correct. Then, click **Save destination**.
+![add jira destination](../../../images/error-inbox/add-jira-destination.png)
+</Step>
+<Step>
+
+Next, set up a template. Click the jira integration icon against the error group and select **Add New Template +**.
+![add new template](../../../images/error-inbox/add-new-template.png)
+</Step>
+<Step>
+
+Add **Jira destination**, **Project**, and **Issue type** on the next page and click **Update message** to save the template. 
+![edit template](../../../images/error-inbox/edit-template.png)
+
+<Callout variant='tip'>
+
+Here, you can also click **Send test notification** to preview the ticket in your jira board before saving the template. Note that a test notification will create a jira ticket in your workspace.
+</Callout>
+</Step>
+</Steps>
+
+Now you can create jira ticket by clicking the jira integration icon on the far right side of the error group and selecting a template.
```

Comment:
I would also mention the other integrations here as well: CodeStream, Slack, etc.